server/datastore/mysql/locks.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/fleetdm/fleet/v4/server/fleet"
 )
 
+var innodbLockWaitsTableExists *bool
```

Comment:
We should use https://pkg.go.dev/sync/atomic#Bool (to avoid races with this global)

Reply:
I switched to atomic var. But I don't think it is mandatory here because all threads should set the var to the same value.

Reply:
As far as I can see it's still a race:
```go
package main

import (
        "sync"
)

var boolVar *bool

func main() {
        var g sync.WaitGroup
        for i := 0; i < 10; i++ {
                g.Add(+1)
                go func() {
                        defer g.Done()
                        if boolVar == nil {
                                val := true
                                boolVar = &val
                        }
                }()
        }
        g.Wait()
}
```
```sh
go run -race ./cmd/some/main.go
==================
WARNING: DATA RACE
Read at 0x000001f923e0 by goroutine 8:
  main.main.func1()
      /Users/luk/fleetdm/git/fleet/cmd/some/main.go:15 +0x7c

Previous write at 0x000001f923e0 by goroutine 5:
  main.main.func1()
      /Users/luk/fleetdm/git/fleet/cmd/some/main.go:17 +0xb0

Goroutine 8 (running) created at:
  main.main()
      /Users/luk/fleetdm/git/fleet/cmd/some/main.go:13 +0x56

Goroutine 5 (finished) created at:
  main.main()
      /Users/luk/fleetdm/git/fleet/cmd/some/main.go:13 +0x56
==================
Found 1 data race(s)
exit status 66
```